### PR TITLE
Server

### PR DESCRIPTION
**Pull request by shaima96**

### CODE REVIEW

File: src/views/layouts/main.hbs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <title>Tutorials</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  <link href="//fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i" rel="stylesheet">
+  <link href="styles.css" rel="stylesheet">
```
Comment (Hadeel-Salamin) [inline src/views/layouts/main.hbs]: correct this to style.css please :3

File: src/server.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const app = require('./app.js')
+
+const port = 3001
```
Comment (Hadeel-Salamin) [inline src/server.js]: for heroku we need to edit this line
const port = process.env.PORT || 3001

File: package.json [hunk 3]
```diff
@@ -17,7 +21,16 @@
...
     "env2": "^2.2.2",
+    "eslint": "^5.10.0",
+    "express": "^4.16.4",
+    "express-handlebars": "^3.0.0",
     "pg": "^7.7.1"
+  },
+  "devDependencies": {
+    "nodemon": "^1.18.9",
+    "tap-spec": "^5.0.0",
+    "tape": "^4.9.1"
```
Comment (shahenazmonia) [inline package.json]: ```tape``` module should be in Dependencies if you want to run an online CI tools

File: package.json [hunk 4]
```diff
@@ -17,7 +21,16 @@
   },
   "homepage": "https://github.com/FACK1/fid-sh-had-week8#readme",
   "dependencies": {
+    "compression": "^1.7.3",
     "env2": "^2.2.2",
+    "eslint": "^5.10.0",
```
Comment (shahenazmonia) [inline package.json]: ```eslint``` should be inside devDependencies. It's not required to run it in production mode

File: src/app.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
+const express=require("express");
+const path=require("path");
+const fs=require("fs");
```
Comment (shahenazmonia) [inline src/app.js]: Why did you call```fs``` module here?

Comment (shaima96) [reply src/app.js]: we fixed @shahenazmonia

**Pull request merged**